packages/dialtone-tokens/themes/config.js

```diff
@@ -1,17 +1,27 @@
 /**
- * Set the current theme and optionally the brand
+ * Set the current theme, brand, and optionally contrast - BACKWARD COMPATIBLE
```

Comment:
Please gutcheck. I'm fairly confident this is backwards-compatible with the third contrast param introduced.